### ISSUE TITLE
glibc's LC_TIME=POSIX strftime "%c" not parsed ("Tue 2020-05-05 01:02:03 UTC")

### ISSUE THREAD

**Issue by charles-dyfis-net** [araddon/dateparse#102]
```
$ LC_CTIME=POSIX date +%c  # on a system using glibc 2.27
Tue 05 May 2020 08:33:47 PM UTC
$ ./dateparse 'Tue 05 May 2020 08:33:47 PM UTC'
fatal: parsing time "Tue 05 May 2020 08:33:47 PM UTC" as "Jan 02 May 2020 08:33:47 PM UTC": cannot parse "Tue 05 May 2020 08:33:47 PM UTC" as "Jan"
```

This impacts the ability to parse timestamps generated by numerous pieces of system software on Linux which use `strftime()` with `%c`.

**Comment by charles-dyfis-net** [comment]
Easiest way to fix this is probably to skip the weekday. `05 May 2020 08:33:47 PM UTC` parses fine.

**Comment by araddon** [comment]
closed with #103  thank you
